tutorials/lava/lib/dl/netx/pilotnet_snn/dataloader.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+from typing import Iterable
+import numpy as np
+import random
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import OutPort, RefPort
+from lava.magma.core.process.variable import Var
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyOutPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import HostCPU
+from lava.magma.core.decorator import implements, requires, tag
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+
+from lava.magma.core.run_configs import Loihi1SimCfg
+from lava.magma.core.run_conditions import RunSteps
+
+
+class State(AbstractProcess):
+    def __init__(
+        self,
+        dataset: Iterable,
+        # set_var: Var,
```

Comment:
Remove.

Reply:
This file should not have been there. Removed it.